Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1243,10 +1220,10 @@ final class StatusTableViewController: ChartsTableViewController {
             hudView.loopCompletionHUD.addGestureRecognizer(statusTapGestureRecognizer)
             hudView.loopCompletionHUD.accessibilityHint = NSLocalizedString("Shows last loop error", comment: "Loop Completion HUD accessibility hint")
             
-            let pumpStatusTapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(pumpStatusTapped(_:)))
+            let pumpStatusTapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(deviceStatusTapped(_:)))
```

Comment:
[nit] Why get rid of the separate `Tapped` handlers for CGM and pump when down below in `deviceStatusTapped` you just have to figure out whether it was CGM or Pump again (e.g. via `sender.view as? CGMStatusHUDView`)?

Reply:
I wanted to reduce code, but thinking about it more, I'll separate the execution of the action from the gathering of the action.